modules/administration-guide/pages/specifying-the-update-approval-strategy.adoc

```diff
@@ -4,28 +4,28 @@
 :page-aliases: installation-guide:upgrading-che-using-operatorhub
 
 [id="specifying-the-{prod-id-short}-operator-update-approval-strategy-using-the-web-console_{context}"]
-= Specifying the {prod-short} operator update approval strategy using the web console
+= Specifying the update approval strategy for the {prod} Operator
 
 
-The {prod-short} operator supports two upgrade strategies:
+The {prod} Operator supports two upgrade strategies:
 
 `Automatic`::
 The Operator installs new updates when they become available.
 
 `Manual`::
 New updates need to be manually approved before installation begins.
 
-You can specify the {prod-short} operator update approval strategy using the OpenShift web console.
+You can specify the update approval strategy for the {prod} Operator by using the OpenShift web console.
 
 .Prerequisites
 
-* A web console session with administrative access on the destination OpenShift instance. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/web_console/web-console.html[Accessing the web console].
+* An OpenShift web console session by a cluster administrator. See link:https://docs.openshift.com/container-platform/{ocp4-ver}/web_console/web-console.html[Accessing the web console].
```

Comment:
The previous formulation is consistently used in our docs.

Reply:
Consider keeping consistency (either way).

Reply:
Thank you. I've grepped the repo and updated it with the same change in two more files.